S/SDL2_ttf/build_tarballs.jl

```diff
@@ -3,20 +3,19 @@
 using BinaryBuilder
 
 name = "SDL2_ttf"
-version = v"2.0.15"
+version = v"2.0.18"
 
 # Collection of sources required to build SDL2_ttf
 sources = [
-    ArchiveSource("https://www.libsdl.org/projects/SDL_ttf/release/SDL2_ttf-$(version).tar.gz",
-                  "a9eceb1ad88c1f1545cd7bd28e7cbc0b2c14191d40238f531a15b01b1b22cd33"),
+    GitSource("https://github.com/libsdl-org/SDL_ttf.git", "3e702ed9bf400b0a72534f144b8bec46ee0416cb"),
     DirectorySource("./bundled"),
 ]
 
 version = v"2.0.16" # <-- this version number is a lie to build for Julia v1.6
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/SDL2_ttf-*/
+cd $WORKSPACE/srcdir/SDL*/
```

Comment:
Maybe
```suggestion
cd $WORKSPACE/srcdir/SDL2_ttf*/
```
so it'll work with both git clones and release tarballs.